docs/odgi_stats.adoc

```diff
@@ -0,0 +1,107 @@
+= odgi stats(1)
+Erik Garrison
+:doctype: manpage
+:release-version: 0.3
+:man manual: odgi stats https://github.com/vgteam/odgi
+:man source: odgi {release-version}
+:page-layout: base
+
+== NAME
+
+odgi_stats -  metrics describing variation graphs
+
+== SYNOPSIS
+
+*odgi stats* [*-i, --idx*=_FILE_] [_OPTION_]...
+
+== DESCRIPTION
+
+The odgi stats(1) command produces statistics of a variation graph. Among other metrics, it can calculate the #nodes, #edges, #paths and the total nucleotide length of the graph. Various histogram summary options complement the tool. If [*-B, --bed-multicov*=_BED_] is set, the metrics will be produced for the intervals specified in the BED.
+
+== OPTIONS
+
+=== Graph Files IO
+
+*-i, --idx*=_FILE_::
+  File containing the succinct variation graph to create statistics from. The file name usually ends with _.og_.
+
+=== Summary Options
+
+*-S, --summarize*::
+  Summarize the graph properties and dimensions. Print to stdout the #nucleotides, #nodes, #edges and #paths of the graph.
+
+*-b, --base-content*::
+  Describe the base content of the graph. Print to stdout the #A, #C, #G and #T of the graph.
+
+*-C, --coverage*::
+  Provide a histogram of path coverage over bases in the graph. Print three columns to stdout: *type*, *cov*, *N*. *type* is one of _full_ or _uniq_ and determines if the histogram corresponds to the full graph or only to a unique paths graph. *cov* implies the #paths. *N* implies the #nucleotides.
+
+*-V, --set-coverage*::
+  Provide a histogram of coverage ofer unique set of paths. Print two columns to stdout: *cov*, *sets*. *cov* implies #nucleotides. *sets* lists the unique set of paths in a comma separated list. Sets with a *cov* of one and no paths in *sets* are listed, too.
+
+*-M, --multi-coverage*::
```

Comment:
@ekg I fail to see the difference between `-V` and `-M`. Would you have an example?

Reply:
-V provides coverage of the _set_ of paths.

-M provides coverage of the _multiset_ (combination with possible repetition of paths).

Reply:
Thanks!